influxql/functions.gen.go

```diff
@@ -284,6 +284,39 @@ func (r *FloatDistinctReducer) Emit() []FloatPoint {
 	return points
 }
 
+// FloatElapsedReducer calculates the elapsed of the aggregated points.
+type FloatElapsedReducer struct {
+	unitConversion int64
+	prev           FloatPoint
+	curr           FloatPoint
+}
+
+// NewFloatElapsedReducer creates a new FloatElapsedReducer.
+func NewFloatElapsedReducer(interval Interval) *FloatElapsedReducer {
+	return &FloatElapsedReducer{
+		unitConversion: int64(interval.Duration),
+		prev:           FloatPoint{Nil: true},
+		curr:           FloatPoint{Nil: true},
+	}
+}
+
+// AggregateFloat aggregates a point into the reducer and updates the current window.
+func (r *FloatElapsedReducer) AggregateFloat(p *FloatPoint) {
+	r.prev = r.curr
+	r.curr = *p
+}
+
+// Emit emits the elapsed of the reducer at the current point.
+func (r *FloatElapsedReducer) Emit() []IntegerPoint {
+	if !r.prev.Nil {
+		elapsed := (r.curr.Time - r.prev.Time) / r.unitConversion
+		return []IntegerPoint{
```

Comment:
I did that to avoid the conflict with the `{{` template operator. Is there a better way?